### ISSUE TITLE
Unit of corners in config file?

### ISSUE THREAD

**Issue by fi6** [bbenligiray/stag#22]
I'm trying to get accurate pose of markers, but I'm not sure what is the unit of the corners in config yaml files. Is it in meter or inch, or something else?

**Comment by bbenligiray** [comment]
I don't know what yaml files you're referring to, but the pose is estimated assuming the top left corner is (0, 0) and the bottom right corner is (1, 1). So 1 unit of distance is equal to the real world width of your printed marker.

**Comment by fi6** [comment]
> I don't know what yaml files you're referring to, but the pose is estimated assuming the top left corner is (0, 0) and the bottom right corner is (1, 1). So 1 unit of distance is equal to the real world width of your printed marker.

Sorry that I may got the wrong repo(should be in stag_ros repo) for the config files. However, when you are saying (0, 0) to (1, 1) for the corners, is 1 unit of distance equal to 1 meter or 1000px, or something different?

**Comment by bbenligiray** [comment]
No, I mean if the black borders of the marker you have printed out is 20cm x 20cm, then 1 unit of distance will be equal to 20cm
